lib/tasks/grade.rake

```diff
@@ -92,6 +97,42 @@ namespace :grade do
 
 end
 
+def sync_specs_with_source
+  reponame = Dir.pwd.split("/").last
+  full_reponame = "appdev-projects/#{reponame}"
+  spec_folder_contents = Octokit.contents(full_reponame, :path => 'spec')
+  spec_subfolder_name = spec_folder_contents.first[:name]
+
+  remote_sha = spec_folder_contents.first[:sha]
+  full_spec_path = "spec/#{spec_subfolder_name}"
+  # Discard unstaged changes in spec folder
+  `git checkout #{full_spec_path} -q`
```

Comment:
What if there are staged changes in `spec/`? 

Admittedly, after significant research, I didn't come up with a clean way to handle this. Perhaps we don't need to worry too much about new files, just modified ones.

Reply:
I think [the previous approach you suggested](https://github.com/firstdraft/grade_runner/pull/57#discussion_r989422516) should work in this case:

```
checkout a temp branch
commit
calculate the SHA
reset
checkout the old branch
delete the temp branch
```

Reply:
Actually, [`git clean` apparently does this](https://stackoverflow.com/a/47014662/10481804). So simply:

```bash
git clean spec
```

should suffice.